### PR TITLE
Feature bug update

### PR DESCRIPTION
**Pull request by sachind-cuelogic**
- Update of adding multiple user from project creation page
- pagination of bug list page
- project name at header

### CODE REVIEW

File: bms/bms_app/static/bms_app/css/common_login_layout.css [hunk 1]
```diff
@@ -30,16 +30,16 @@ body{
...
-	padding: 20px;
+  padding: 20px;
 }
 
 .alreadymember{
-    border-top: 1px solid#888;
-    padding-top:20px; 
-    font-size:15px;
-    text-align: center;
+  border-top: 1px solid#888;
```
Comment (nileshm-cuelogic) [inline bms/bms_app/static/bms_app/css/common_login_layout.css]: add space between "solid" and "#888"

File: bms/bms_app/static/bms_app/js/createbug.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-window.onload = function() {
+/*window.onload = function() {
```
Comment (tusharm-cuelogic) [inline bms/bms_app/static/bms_app/js/createbug.js]: remove unused code.

File: bms/bms_app/views.py [hunk 3]
```diff
@@ -78,13 +78,16 @@ def website_home(request):
 @login_required(login_url='/login/')
 def create_project(request):
 
+    project_name_list = header_sidebar(request)
+    print "project list===>",project_name_list
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: remove print statement

File: bms/bms_app/views.py [hunk 4]
```diff
@@ -119,6 +122,9 @@ def create_project(request):
 @login_required(login_url='/login/')
 def project_list(request):
 
+    project_name_list = header_sidebar(request)
+    print "project list===>",project_name_list
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: remove print statement

File: bms/bms_app/views.py [hunk 5]
```diff
@@ -165,20 +171,23 @@ def delete_project(request):
 def create_bug(request):
     if request.user.is_authenticated():
         current_user = request.user
-    pid=0
+    # pid=0
 
     if request.session.has_key('pid'):
         pid = request.session['pid']
         # print "request session in create bug====>",pid
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: remove print statements

File: bms/bms_app/views.py [hunk 6]
```diff
@@ -165,20 +171,23 @@ def delete_project(request):
...
+
     if request.method == 'GET':
         project_name = ProductUser.objects.all().filter(prod_user_id = current_user.id)
-        if pid == 0:
-            intcount = 0
-            for projectIds in project_name: 
-                if intcount == 0:
-                    pid = projectIds.id
-                intcount += 1 
+        # if pid == 0:
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: remove unused code

File: bms/bms_app/views.py [hunk 7]
```diff
@@ -213,24 +222,34 @@ def create_bug(request):
...
 
     if request.session.has_key('pid'):
         pid = request.session['pid']
         # print "request session in create bug====>",pid
 
-    if request.GET.get('pid'):
-        pid = int(request.GET.get('pid'))
-        # print "bug list pid===>",pid
-        request.session['pid'] = pid
+    # if request.GET.get('pid'):
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: remove unused code

File: bms/bms_app/views.py [hunk 8]
```diff
@@ -213,24 +222,34 @@ def create_bug(request):
...
+    #     pid = int(request.GET.get('pid'))
+    #     # print "bug list pid===>",pid
+    #     request.session['pid'] = pid
         # print "bug list session id==>",request.session['pid']
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: remove unused code and print statemenst

File: bms/bms_app/views.py [hunk 9]
```diff
@@ -338,12 +355,36 @@ def privacy(request):
...
 
 def landing_header_footer(request):
     return render(request, 'registration/landing_header_footer.html')
     
 def get_comments(bid):  
     post_comment = Comments.objects.filter(bug_id=bid)
     return post_comment
+
+
+def header_sidebar(request):
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py:362]: remove unused code

File: bms/bms_app/views.py [hunk 10]
```diff
@@ -228,11 +228,11 @@ def bug_list(request):
         pid = request.session['pid']
         # print "request session in create bug====>",pid
```
Comment (tusharm-cuelogic) [inline bms/bms_app/views.py]: so many unused code and print statements please remove it

File: bms/bms_app/static/bms_app/js/create_prod.js [hunk 11]
```diff
@@ -15,37 +18,56 @@ $(function() {
...
-            $('[data-role="dynamic-fields"] > .form-inline:first-child [data-role="remove"]').prop("disabled", true);
+            $('[data-role="dynamic-fields"] > .form-inline:first-child [data-role="remove"]').prop("disabled", false);
+            $('[data-role="dynamic-fields"] > .form-inline:first-child select').prop("disabled", false);
            /* console.log("inside enable button");*/
-            $("#modalbody .form-inline button").prop("disabled", false); 
+           
         }
     };
 
     disableCross();
```
Comment (tusharm-cuelogic) [inline bms/bms_app/static/bms_app/js/create_prod.js]: remove unused code

File: bms/bms_app/static/bms_app/js/bug_view.js [hunk 12]
```diff
@@ -41,12 +41,11 @@ $(document).ready(function() {
             url: "../bug_list/",
             success: function(result) 
             {
-                console.log(result);
```
Comment (tusharm-cuelogic) [inline bms/bms_app/static/bms_app/js/bug_view.js]: remove unused code

File: bms/bms_app/static/bms_app/js/file_upload.js [hunk 13]
```diff
@@ -1,63 +1,3 @@
-/*
```
Comment (tusharm-cuelogic) [inline bms/bms_app/static/bms_app/js/file_upload.js]: remove unused code

File: bms/bms_app/static/bms_app/js/prod_ajax.js [hunk 14]
```diff
@@ -28,42 +28,44 @@ $(document).ready(function() {
...
+    var getuser = $('#get_username').text();
+
+    var select=document.getElementById('user-name');
+
+    for (i=0;i<select.length;  i++) {
+       if (select.options[i].value==getuser) {
+         select.remove(i);
+       }
+    }
+
```
Comment (tusharm-cuelogic) [inline bms/bms_app/static/bms_app/js/prod_ajax.js:40]: remove console.log statements

**Pull request merged**